site/docs/guide.md

```diff
@@ -157,6 +157,14 @@ workspace.
   <td><code>//foo/...:all-targets</code></td>
   <td>All targets (rules and files) in all packages beneath the directory <code>foo</code>.</td>
 </tr>
+<tr>
+  <td><code>//...</code></td>
+  <td>All rules in all non-<a href="external.html">external</a> packages.</td>
+</tr>
+<tr>
+  <td><code>//:all</code></td>
+  <td>All rules in the top-level package.</td>
```

Comment:
All targets in the top-level package, if there is a BUILD file at the root of the workspace.